src/models/log-result.ts

```diff
@@ -0,0 +1,6 @@
+import { RiskPolicy } from './risk-policy';
+import { Verdict } from './verdict';
+
+export type LogResult = {
+
```

Comment:
let's allow for a flexible response, without the fixed structure like it's done for "authenticate" action for example